routes/web.php

```diff
@@ -106,147 +106,275 @@
     });
 
     Route::get('workbench', function() {
-        return redirect('https://talk.restarters.net/t/our-work-on-repair-data/1150');
+        if (config('restarters.features.discourse_integration')) {
+            return redirect('https://talk.restarters.net/t/our-work-on-repair-data/1150');
+        } else {
+            return redirect()->route('home');
```

Comment:
This seems fine, but we can handle this in a bit less of a messy or duplicative way.

First question, do we need to set the 307 back to the homepage? Should these routes not just 404?

Also, we can split out a separate file of routes, and import them wholesale, and conditionally. That could significantly cut down the noise.

Reply:
Addressed in [35dcd05](https://github.com/iFixit/restarters/pull/11/commits/35dcd0538a08a2a55d53d75d9ce4fd9d16fdf684)

We can actually just drop this whole block.